components/dialog/test/dialog.visual-diff.js

```diff
@@ -44,34 +44,40 @@ describe('d2l-dialog', () => {
 					});
 
 					it('opened', async function() {
-						await helper.open(page, '#dialog');
+						await page.$eval('#dialog', (dialog) => dialog.opened = true);
 						await visualDiff.screenshotAndCompare(page, this.test.fullTitle());
 					});
 
 					it('scroll bottom shadow', async function() {
-						await helper.open(page, '#dialogLong');
+						await page.$eval('#dialogLong', (dialog) => dialog.opened = true);
 						await visualDiff.screenshotAndCompare(page, this.test.fullTitle());
 					});
 
 					it('scroll top shadow', async function() {
-						await helper.open(page, '#dialogLong');
-						await page.$eval('#dialogLong #bottom', (bottom) => {
-							bottom.scrollIntoView();
+						await page.$eval('#dialogLong', (dialog) => {
```

Comment:
This is another example that I think is actually better waiting for the event, otherwise this code needs to wait a frame before trying to scroll.